playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: Print list of known health checks
+    action: openshift_health_check
+    when: checks is undefined or not checks
```

Comment:
To print a list of known checks:

```
$ ansible-playbook -i <inventory file> playbooks/byo/openshift-checks/adhoc.yml
```

But that's not documented (yet). The problem with the current output is that it repeats for as many hosts as you have in the `OSEv3` group.

I would like to de-duplicate error messages by grouping together hosts that had the same error.